packages/presentation/presentation_publishing/interfaces/fetch/publishes_reload.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { PublishingSubject } from '../../publishing_subject';
+
+export interface PublishesReload {
```

Comment:
nit: I think this interface could be extended with a `reload` function as well. This way, we can expose the ability to imperatively trigger a reload.

Reply:
That makes sense but I think we can hold off adding interface and implementing in dashboard until its needed.